v2/internal/pkg/release/cincinnati.go

```diff
@@ -128,6 +129,22 @@ func (o *CincinnatiSchema) GetReleaseReferenceImages(ctx context.Context) ([]v2a
 			Origin:      o.Config.Mirror.Platform.Release,
 		}
 		allImages = append(allImages, copyImage)
+		// OCPBUGS-50617
+		// include signature verify and download for rc (release candidate) by digest
+		// skip ec (engineering candidate) and other releases (by tag)
+		imgSpec, err := image.ParseRef(o.Config.Mirror.Platform.Release)
+		if err != nil {
+			return []v2alpha1.CopyImageSchema{}, err
+		}
+		if imgSpec.IsImageByDigestOnly() {
+			imgs, err := o.Signature.GenerateReleaseSignatures(ctx, allImages)
+			if err != nil {
+				return []v2alpha1.CopyImageSchema{}, err
+			}
+			return imgs, nil
+		} else {
+			o.Log.Warn("release image is not by digest: SKIPPING signature verification")
+		}
```

Comment:
Hi @lmzuccarelli
So just to understand, we would be saying that for releases passed on directly with the `release` stanza, oc-mirror will not verify signatures, nor provide the signature configMap? 
:
```
mirror:
  platform:
    release: quay.io/openshift-release-dev/ocp-release@:v4.16.0
```

For EC and RC, this might not be a big problem... Unless they want to test this on a disconnected environment, in which case, they still need the configMap, no?

Also, what if this were used by an end user requesting a specific release (not candidate nor RC), like in the above example?

We could get the digest from the tag by using `GetDigest` from the manifest package.